apps/condo/domains/resident/tasks/notifyResidentsOnPayday.js

```diff
@@ -41,7 +40,7 @@ async function sendNotification (context, receipt, consumer) {
     const resident = consumer.resident
     const userId = resident.user.id
     const lang = get(COUNTRIES, [get(receipt, 'organization.country', conf.DEFAULT_LOCALE), 'locale'], conf.DEFAULT_LOCALE)
-    const now = dayjs().subtract(1, 'month')
+    const monthName = lang === RU_LOCALE ? dayjs(receipt.period).locale('ru').format('MMMM') : dayjs(receipt.period).format('MMMM')
```

Comment:
Why not just pass this parameter inside the `dayjs.locale` function?
```suggestion
    const monthName = dayjs(receipt.period).locale(RU_LOCALE).format('MMMM')
```